content/css/docs.css

```diff
@@ -292,7 +292,7 @@ footer {
   padding: 5px 20px 20px 7px;
 }
 
-.side-nav .fa {
+.side-nav .ti {
```

Comment:
Could you also add `font-size: 1.1em` here? I feel like the new icons are a bit small compared to the text at their regular `1em` size.